_docs/howto/deploy/change-gcp-deployment.md

```diff
@@ -8,13 +8,16 @@ Note: If you used the Forseti setup wizard to deploy, your `deploy-forseti.yaml`
 will have a timestamp suffix, e.g. `deploy-forseti-20171001000000.yaml`.
 
 ### Get a particular deployment template version
-This step is **optional**.
+-This step is **optional**.
 
 If you need to change the `release-version` for your Forseti deployment, you MUST 
 get the correct version of Forseti. The deployment template's startup 
 script has release-specific code, so things will break if you use a startup script that 
 is out of sync with the deployed release.
 
+You need to be where you previously retrieved the code from GitHub
```

Comment:
What does this bit mean? I have to use the same tool and I have to be in the same directory?

Reply:
This is to say that users have to be at the local repo where they had run their previous deployment.  It does seem not needed, so I'll drop it.